pkg/operator/controllers/image/image_controller.go

```diff
@@ -0,0 +1,142 @@
+package image
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"context"
+
+	"github.com/Azure/go-autorest/autorest/azure"
+	configv1 "github.com/openshift/api/config/v1"
+	configclient "github.com/openshift/client-go/config/clientset/versioned"
+	"github.com/sirupsen/logrus"
+	"github.com/ugorji/go/codec"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/builder"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/predicate"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+
+	aroclient "github.com/Azure/ARO-RP/pkg/operator/clientset/versioned"
+	"github.com/Azure/ARO-RP/pkg/operator/controllers"
+)
+
+const imageConfigResource = "cluster"
+
+var allowedRegistries = []string{}
+
+type Reconciler struct {
+	arocli     aroclient.Interface
+	configcli  configclient.Interface
+	log        *logrus.Entry
+	jsonHandle *codec.JsonHandle
+}
+
+func NewReconciler(log *logrus.Entry, arocli aroclient.Interface, configcli configclient.Interface) *Reconciler {
+	return &Reconciler{
+		arocli:     arocli,
+		log:        log,
+		jsonHandle: new(codec.JsonHandle),
+		configcli:  configcli,
```

Comment:
nit: keep struct order the same as it was defined
```
arocli: 
configcli:    
log:       
jsonHandle:

Reply:
Thanks Ross! Fixed